applications/bookmyshow/lib/loggers.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+
+
+
+logging.basicConfig(format='%(asctime)s::%(filename)s::%(lineno)s::%(funcName)s\
+                    ::%(msg)s', level =logging.INFO , filename='home_info.log', filemode='a')
+
+logger = logging.getLogger(__name__)
+class Logger():
+
+    def __init__(self,logger =None):
+        self.logger = logger or logging.getLogger(__name__)
+
+    def log_info(self, msg):
```

Comment:
Please define all methods as class methods instead of instance method and avoid to create a object of Logger class in different places